charts/das/README.md

```diff
@@ -285,13 +285,7 @@ Option | Description | Default
 `data-availability.disable-signature-checking` | disables signature checking on Data Availability Store requests (DANGEROUS, FOR TESTING ONLY) | None
 `data-availability.enable` | enable Anytrust Data Availability mode | `true`
 `data-availability.extra-signature-checking-public-key` | string                               public key to use to validate Data Availability Store requests in addition to the Sequencer's public key determined using sequencer-inbox-address, can be a file or the hex-encoded public key beginning with 0x; useful for testing | None
-`data-availability.ipfs-storage.enable` | enable storage/retrieval of sequencer batch data from IPFS | None
-`data-availability.ipfs-storage.peers` | strings                                               list of IPFS peers to connect to, eg /ip4/1.2.3.4/tcp/12345/p2p/abc...xyz | None
-`data-availability.ipfs-storage.pin-after-get` | pin sequencer batch data in IPFS | `true`
-`data-availability.ipfs-storage.pin-percentage` | float                                        percent of sequencer batch data to pin, as a floating point number in the range 0.0 to 100.0 | `100`
-`data-availability.ipfs-storage.profiles` | string                                             comma separated list of IPFS profiles to use, see https://docs.ipfs.tech/how-to/default-profile | None
-`data-availability.ipfs-storage.read-timeout` | duration                                       timeout for IPFS reads, since by default it will wait forever. Treat timeout as not found | `1m0s`
-`data-availability.ipfs-storage.repo-dir` | string                                             directory to use to store the local IPFS repo | None
+`data-availability.ipfs-storage.enable` | legacy option - not supported | None
```

Comment:
Because this feature is being deprecated, lets also remove all the ipfs related stuff in the das chart. https://github.com/OffchainLabs/community-helm-charts/blob/230644dfdfd62db422184b26d62429a388e75880/charts/das/values.yaml#L244-L251

Reply:
Removed, also updated deafult log level